libmscore/barline.cpp

```diff
@@ -1272,10 +1272,11 @@ QString BarLine::accessibleExtraInfo() const
                   for (const Element* e : nextM->el()) {
                         if (!score()->selectionFilter().canSelect(e))
                               continue;
-                        if (e->isMarker())
+                        if (e->isMarker()) {
                               if (toMarker(e)->markerType() == Marker::Type::FINE)
                                     continue; //added above^
                               rez = QString("%1 %2").arg(rez).arg(e->screenReaderInfo());
+                              }
```

Comment:
From what I remember, I wanted the screen reader to say the Markers and Jumps as extraInfo when a barline was selected. I think that the proposed change fixes a bug of mine.

@MarcSabatella: I wouldn't remove the loop. In my opinion, even if at some point there would be a command for navigating all elements directly, my idea was to have the following workflow, based on how I've seen people read scores in general (to be clear: not visually impaired musicians):
- The user uses the existing command to familiarise with the main elements of the score (Clefs, Time Signatures, Notes, Rests, Barlines, etc.) and he can skip the extra info.
- After he has a good idea about the melody and rhythm, he can traverse the score again and listen to the extra info, meaning elements that are attached.
- And with a new command the user can have the possibility to select each individual element, but I was thinking that is mainly for editing.

In my opinion it is more natural to have the screen reader say all the elements that are attached to the selected one as extra info, instead of having to select each element and figure out to what they are attached to based on bar and beat. I would be best to have some feedback from users on this.